src/composable/ConditionalOrder.ts

```diff
@@ -281,6 +286,43 @@ export abstract class ConditionalOrder<D, S> {
         []
       )
 
+      let orderBookApi = orderBookCache[chainId]
+      if (!orderBookApi) {
+        orderBookApi = new OrderBookApi({ chainId })
+        orderBookCache[chainId] = orderBookApi
+      }
+
+      // TODO: Derive orderId from the order data
+      /*
+      {
+        name: "Gnosis Protocol",
+        version: "v2",
+        chainId: chainId,
+        verifyingContract: GPV2SETTLEMENT,
+      }
+      */
+
+      const orderUid = await computeOrderUid(chainId, owner, order as Order)
```

Comment:
I don't get the TODO. Aren't you doing that on line 305?

Reply:
Forgot to remove that! done